package.json

```diff
@@ -4,7 +4,7 @@
   "description": "WYSIWYG-like EmojiOne Converter / Picker Plugin for jQuery",
   "main": "dist/emojionearea.js",
   "scripts": {
-    "build": "node_modules/.bin/grunt",
+    "build": "npm install && grunt",
```

Comment:
I didn't realize this was already in here when creating the PR I made.. Perhaps, adding a script just for grunt could be nice too? There's no point to run `npm install` every time we just want to rebuild the files... :thinking:

Reply:
You are right. I probably made some confusion.